src/iadmin.cpp

```diff
@@ -1672,11 +1581,6 @@ doCommand( char *cmdToken[], rodsArguments_t* _rodsArgs = 0 ) {
         generalAdmin( 0, "rm", "resource", cmdToken[1], cmdToken[2], cmdToken[3], cmdToken[4], cmdToken[5], cmdToken[6], "", "", _rodsArgs );
         return 0;
     }
-    if ( strcmp( cmdToken[0], "rmdir" ) == 0 ) {
-        generalAdmin( 0, "rm", "dir", cmdToken[1], cmdToken[2],
-                      cmdToken[3], cmdToken[4], cmdToken[5], cmdToken[6], "", "" );
-        return 0;
-    }
```

Comment:
Are we mainly interested in just removing `iadmin`'s exposure of this functionality, or the whole feature? The "rm" of a "dir" is apparently part of the GeneralAdmin API.

Reply:
i think the exposure for now.

regarding the whole feature... having an administrator side door that potentially/definitely doesn't go through policy feels... weird.

Reply:
Agreed. As a result of this PR, the removed subcommands are deprecated. The deprecation can be captured in the release notes. Just need to open issues for removing the subcommands in the 6.0 milestone.